tools/fuchsia/build_fuchsia_artifacts.py

```diff
@@ -200,29 +199,55 @@ def ProcessCIPDPackage(upload, engine_version):
   cipd_yaml = os.path.join(_script_dir, 'fuchsia.cipd.yaml')
   CopyFiles(cipd_yaml, os.path.join(_bucket_directory, 'fuchsia.cipd.yaml'))
 
-  tag = 'git_revision:%s' % engine_version
-  already_exists = CheckCIPDPackageExists('flutter/fuchsia', tag)
-  if already_exists:
-    print('CIPD package flutter/fuchsia tag %s already exists!' % tag)
+  # Copy the license files from the source directory to be next to the bucket we
+  # are about to package.
+  bucket_root = os.path.join(_bucket_directory, 'flutter')
+  licenses_root = os.path.join(_src_root_dir, 'flutter/ci/licenses_golden')
+  license_files = [
+    'licenses_flutter',
+    'licenses_fuchsia',
+    'licenses_gpu',
+    'licenses_skia',
+    'licenses_third_party'
+  ]
+  for license in license_files:
+    src_path = os.path.join(licenses_root, license)
+    dst_path = os.path.join(bucket_root, license)
+    CopyPath(src_path, dst_path)
 
-  if upload and IsLinux() and not already_exists:
-    command = [
-        'cipd', 'create', '-pkg-def', 'fuchsia.cipd.yaml', '-ref', 'latest',
-        '-tag',
-        tag,
-    ]
-  else:
+  if not upload or not IsLinux():
     command = [
         'cipd', 'pkg-build', '-pkg-def', 'fuchsia.cipd.yaml', '-out',
         os.path.join(_bucket_directory, 'fuchsia.cipd')
     ]
+    subprocess.check_call(command, cwd=_bucket_directory)
```

Comment:
Do we still need the retries?

Reply:
I don't think so -- cipd pkg-build is local-only so it doesn't have the same chance to fail as cipd create does

Reply:
I ended up keeping the fallbacks anyways